pkg/security/module/module.go

```diff
@@ -172,15 +190,15 @@ func (m *Module) Reload() error {
 	ruleSet := m.probe.NewRuleSet(newRuleSetOpts())
 
 	loadErr := rules.LoadPolicies(m.config.PoliciesDir, ruleSet)
-	if loadErr.ErrorOrNil() != nil {
-		log.Errorf("error while loading policies: %+v", loadErr.Error())
-	}
 
 	model := &model.Model{}
 	approverRuleSet := rules.NewRuleSet(model, model.NewEvent, newRuleSetOpts())
-	loadErr = rules.LoadPolicies(m.config.PoliciesDir, approverRuleSet)
+	loadApproversErr := rules.LoadPolicies(m.config.PoliciesDir, approverRuleSet)
```

Comment:
Line 232, `monitor.ReportRuleSetLoaded(ruleSet, loadErr)` now use the first load multierror instead of the second. Should they always return the same errors ?

Reply:
Both should return the same errors but in fact only the first one is "fatal"